client/blocks/login/login-form.jsx

```diff
@@ -545,6 +548,48 @@ export class LoginForm extends Component {
 		);
 	}
 
+	handleMagicLoginClick = () => {
+		this.props.recordTracksEvent( 'calypso_login_magic_login_request_click', {
+			origin: 'login-form',
+		} );
+		this.props.resetMagicLoginRequestForm();
+	};
+
+	renderMagicLoginLink() {
+		if (
+			! canDoMagicLogin(
+				this.props.twoFactorAuthType,
+				this.props.oauth2Client,
+				this.props.wccomFrom,
+				this.props.isJetpackWooCommerceFlow
+			)
+		) {
+			return null;
+		}
+
+		const loginLink = getLoginLinkPageUrl(
+			this.props.locale,
+			this.props.currentRoute,
+			this.props.currentQuery?.signup_url,
+			this.props.oauth2Client?.id
+		);
+		const { query, usernameOrEmail } = this.props;
+		const emailAddress = usernameOrEmail || query?.email_address || this.state.usernameOrEmail;
+
+		const magicLoginPageLinkWithEmail = addQueryArgs( { email_address: emailAddress }, loginLink );
+
+		return this.props.translate(
+			' Would you like to {{magicLoginLink}}receive a login link{{/magicLoginLink}}?',
```

Comment:
You already have a space before this call. 
```suggestion
			'Would you like to {{magicLoginLink}}receive a login link{{/magicLoginLink}}?',
```

Reply:
You're perfectly right!